datagateway_api/src/common/filters.py

```diff
@@ -27,11 +27,23 @@ def __init__(self, field, value, operation):
         self.value = value
         self.operation = operation
 
-        if self.operation == "in":
+        if self.operation in ["in", "nin", "inq"]:
             if not isinstance(self.value, list):
                 raise BadRequestError(
-                    "When using the 'in' operation for a WHERE filter, the values must"
-                    " be in a list format e.g. [1, 2, 3]",
+                    f"When using the {self.operation} operation for a WHERE filter, the"
+                    f" values must be in a list format e.g. [1, 2, 3]",
```

Comment:
If making the change suggested below, you might want to change the example list to `[1, 2]` to match that a between should only have two values given to it.